Formula/q.rb

```diff
@@ -6,7 +6,7 @@ class Q < Formula
   url "https://github.com/harelba/q/archive/2.0.19.tar.gz"
   sha256 "cd4c60923bc40f53d974b54849f76096bf9901407c618cd0a3ccbc322aacc97d"
   license "GPL-3.0-or-later"
-  revision 2
+  revision 3
```

Comment:
Forgot to bump revision in previous PR https://github.com/Homebrew/homebrew-core/pull/80043/files